build.gradle

```
@@ -21,17 +21,23 @@ repositories {
   jcenter()
 }
 
+java {
+  toolchain {
+    languageVersion = JavaLanguageVersion.of(11)
+  }
+}
+
 dependencies {
   // JUnit Jupiter API for testing.
-  testImplementation 'org.junit.jupiter:junit-jupiter-api:5.5.2'
+  testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
 
   // JUnit Jupiter Engine for testing.
-  testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.5.2'
+  testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
```

```suggestion
  testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
```

In the examples from Gradle they don't include a version here. I assume its automatically getting the correct version because of the version specified for `org.junit.jupiter:junit-jupiter-api`.

---

It obviously depends on which examples you look at; [the JUnit documentation](https://junit.org/junit5/docs/current/user-guide/#running-tests-build-gradle) includes a version for both.

I assume [this example](https://docs.gradle.org/current/userguide/java_testing.html#compiling_and_executing_junit_jupiter_tests) is the one you're referring to? I'll go ahead and remove the second version since it avoids duplicating a piece of data, but I couldn't find any documentation that explicitly explained why that should work.

---

Yeah I wouldn't mind leaving it as I also didn't find any stated reason that should work, I was just assuming there is some kind of peer dependency magic going on.